src/Utils.js

```diff
@@ -0,0 +1,52 @@
+import $ from 'jquery';
+
+const proxy = 'https://cors.now.sh/';
+const maxYear = 2050;
+const minYear = 1950;
+const defaultYear = (new Date()).getFullYear();
+
```

Comment:
Consider moving these kind of defaults/constants to a config.js file so you have it centralized in one place in case you need to change it.